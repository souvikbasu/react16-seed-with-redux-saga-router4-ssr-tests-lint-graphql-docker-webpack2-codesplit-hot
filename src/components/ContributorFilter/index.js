import React from 'react'
import FilterLink from '../../containers/FilterLink'

const Footer = () => (
  <p>
    Show:
    {' '}
    <FilterLink filter="SHOW_ALL">
      All
    </FilterLink>
    {', '}
    <FilterLink filter="SHOW_TOP_CONTRIBUTORS">
      Top Contributors
    </FilterLink>
  </p>
)

export default Footer