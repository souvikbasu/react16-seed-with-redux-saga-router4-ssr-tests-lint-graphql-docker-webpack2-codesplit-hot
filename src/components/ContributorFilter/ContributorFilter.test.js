import React from 'react';
import { shallow } from 'enzyme';

import ContributorFilter from './';

describe('<ContributorFilter />', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ContributorFilter />);
    expect(wrapper).toMatchSnapshot();
  });
});
