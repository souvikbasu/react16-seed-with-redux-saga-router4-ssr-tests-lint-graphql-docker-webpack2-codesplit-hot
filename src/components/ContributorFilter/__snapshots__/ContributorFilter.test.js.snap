// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContributorFilter /> matches snapshot 1`] = `
<p>
  Show: 
  <FilterLink
    filter="SHOW_ALL"
  >
    All
  </FilterLink>
  , 
  <FilterLink
    filter="SHOW_TOP_CONTRIBUTORS"
  >
    Top Contributors
  </FilterLink>
</p>
`;
